sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -171,7 +173,7 @@ export class IdentityClient extends ServiceClient {
 
   static getDefaultOptions(): TokenCredentialOptions {
     return {
-      authorityHost: DefaultAuthorityHost
+      authorityHost: envAuthorityHost || DefaultAuthorityHost
```

Comment:
Do we want to change the default options based on something in the environment? That feels like it might confuse users.